ai-gateway/concepts/api-keys.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Provider API Keys
+description: Understanding provider API key management in the AI Gateway
+---
+
+Provider API keys authenticate requests to AI providers like OpenAI, Anthropic, and Google. The AI Gateway offers flexible key management—from simple passthrough to centralized management with automatic failover.

```

Comment:
suggestion: `The AI Gateway` -> `An ngrok AI Gateway` or `The AI Gateway action`